java/src/org/certificatetransparency/ctlog/LogEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class LogEntry {
+  public Ct.X509ChainEntry x509Entry;
```

Comment:
The end state is to not require `protobuf.jar` or `protoc` to build the Java side.